test/java/com/arpnetworking/metrics/portal/reports/impl/chrome/BaseChromeTest.java

```diff
@@ -40,11 +42,12 @@
  *
  * @author Spencer Pearson (spencerpearson at dropbox dot com)
  */
-public class BaseChromeIT {
+public class BaseChromeTest {
```

Comment:
Should not end with `IT` or `Test` if this class does not actually include tests. Further, it should probably be marked `abstract` if it is intended for extension/code sharing only.